File: PHP/Product.php
<?php

class Product{
    private $idProduct;
    private $price;

    public function __construct($id = "", $p = 0){
        $this->idProduct = $id;
        $this->price = $p;
    }

    public function setIDProduct($id){
        $this->idProduct = $id;
    }

    public function getIDProduct(){
        return $this->idProduct;
    }
    
    public function setPrice($p){
        $this->price = $p;
    }

    public function getPrice(){
        return $this->price;
    }
}
?>

File: CPP/Product.cpp
class Product{
    private:
        string idProduct;
        int price;

    public:
        Product(){
        }

        Product(string id, int p){
            this->idProduct = id;
            this->price = p;
        }

        void setIDProduct(string id){
            this->idProduct = id;
        }
        string getIDProduct(){
            return this->idProduct;
        }

        void setPrice(int p){
            this->price = p;
        }
        int getPrice(){
            return this->price;
        }

        ~Product(){
        }
};